src/SearchHeadlessProvider.tsx

```diff
@@ -1,33 +1,14 @@
 import { PropsWithChildren } from 'react';
-import { provideHeadless, SearchHeadless, HeadlessConfig } from '@yext/search-headless';
+import { SearchHeadless } from '@yext/search-headless';
 import { SearchHeadlessContext } from './SearchHeadlessContext';
-import acquireSessionId from './utils/acquireSessionId';
-import packageJson from '../package.json';
 
-const { version } = packageJson;
-
-type Props = HeadlessConfig & {
-  verticalKey?: string,
-  sessionTrackingEnabled?: boolean
-};
+type Props = { searcher: SearchHeadless };
 
 export function SearchHeadlessProvider(props: PropsWithChildren<Props>): JSX.Element {
-  const { children, verticalKey, sessionTrackingEnabled=true, ...searchConfig } = props;
-  const additionalHttpHeaders = {
-    'Client-SDK': {
-      ANSWERS_HEADLESS_REACT: version
```

Comment:
if we don't need to pass this header anymore, we could simplify `provideHeadless` and remove the overloaded signatures in Headless

Reply:
do you remember if we said that users are not allowed to set additionalHttpHeaders? I vaguely remember that being the point of the overloaded signatures but can't remember exactly

Reply:
yes, that's why we had the overloaded signatures and marked the one that accepted headers as `@internal`. we didn't want to support users adding their own arbitrary headers

Reply:
I'll root through the Slack convos and see if there were other use cases for the additional headers!

Reply:
Cool! Sounds like we can simplify Headless then!